be/compendium/db/migrations/0001_initial.py

```diff
@@ -1,37 +1,58 @@
-# Generated by Django 3.0.14 on 2022-08-08 11:02
+# Generated by Django 3.0.14 on 2022-08-15 07:27
 
+from django.conf import settings
+import django.contrib.auth.models
+import django.contrib.auth.validators
 from django.db import migrations, models
 import django.db.models.deletion
+import django.utils.timezone
 
 
 class Migration(migrations.Migration):
 
     initial = True
 
     dependencies = [
+        ('auth', '0011_update_proxy_permissions'),
     ]
 
     operations = [
         migrations.CreateModel(
-            name='Accounts',
+            name='User',
```

Comment:
- may I ask. why this one changed to singular from before `Users`?

Reply:
Django authentication has it's own django.auth.models  sir, and my problem since last week was that I was trying to configure the default auth model of Django and change it to the Users model that we created. And then that's why  tried to debug it, but it required me to add more fields in the model just to satisfy the requirements of all the fields to make the authentication happen, once I did that , there was another error that I don't know where it starts but i know that it's because of extraneous configuration of the Custom model that won't satisfy the Django Authentication to make it as a default Auth_user_model. And I was dealing with it back and forth and since my skillset is limited yet and I am just learning django as I make this project it's been disconcerting to know that I am still in the beginning of my developmental phase. Furthermore if you have a suggestion or a fix to make the Custom Model be as it is and letting Django make it as a Auth-user-model and then authenticate it to add a token so that the data that was registered then I would gladly take your suggestion and apply it in the models and make that as the permanent auth model and not just transitory,
But as of right now this is the only way i know how yet.